packages/material-ui-system/src/index.d.ts

```diff
@@ -262,6 +263,417 @@ export const visuallyHidden: React.CSSProperties;
 
 // eslint-disable-next-line @typescript-eslint/naming-convention
 export function unstable_getThemeValue(prop: string, value: any, theme: object): any;
+
+export type StandardCSSProperties = CSS.PropertiesFallback<number | string>;
+
+/**
+ * The `css` function accepts arrays as values for mobile-first responsive styles.
+ * Note that this extends to non-theme values also. For example `display=['none', 'block']`
+ * will also works.
+ *
+ * For more information see: https://styled-system.com/responsive-styles
+ */
+export type ResponsiveStyleValue<T> = T | Array<T | null> | { [key: string]: T | null };
+
+/**
+ * All non-vendor-prefixed CSS properties. (Allow `number` to support CSS-in-JS libs,
+ * since they are converted to pixels)
```

Comment:
```suggestion
 * All non-vendor-prefixed CSS properties. (Also allows `number` in order to support CSS-in-JS libs,
 * since they are converted to `px`.)
```